### DISCUSSION TITLE
Design

### DISCUSSION THREAD

**Discussion by chrabou** [lucile98-ode/GPI-L2BHA-FENOUIL#31] [Ideas]
Essayer de revoir le design de l'application pour le rendre un peu plus "jolie" et l'accorder au site web.

**Comment by lucile98-ode** [comment]
On peut ajouter une image dans le code pour le fond @amrx12

**Reply by amrx12** [reply]
Oui il faudrait que je regarde ça si j'ai le temps, si vous avez déjà des images mettez les de coté
